enhancements/installer/powervs-ipi.md

```diff
@@ -0,0 +1,237 @@
+---
+title: powervs-ipi
+authors:
+  - "@jaypoulz"
+reviewers:
+  - "@crawford"
+  - "@andymcc"
+  - "@Prashanth684"
+approvers:
+  - TBD
+creation-date: 2021-04-16
+last-updated: 2021-04-16
+status: implementable
+---
+
+# powervs-platform-provider
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document describes how [PowerVS][powervs-website] becomes an infra provider for OpenShift. PowerVS is a cloud offering by IBM that aims to provid customers interested in running secure and flexible workloads on IBM Power (ppc64le) hardware without having to develop and maintain an on premise datacenter. The goal of this enhancement would be to leverage the PowerVS APIs to deploy an OpenShift cluster on this cloud environment. In addition, the APIs would be leveraged for cluster resizing operations post installation.
```

Comment:
I've been talking to @clnperez and others and i think we need to make a distinction between the cloud provider and the platform and make sure that we spell out that IBM Cloud is the provider and the platform is PowerVS. In supporting PowerVS as a platform we might have to implement certain parts of the IBM cloud provider that we need to make sure we explicitly call out.

Reply:
I believe this newest version does a better job of calling that out explicitly. Please review and resolve if this is addressed to your liking.

Reply:
Going to mark this as resolved, and someone can reopen if I missed something important. :)